.github/workflows/ci.yml

```diff
@@ -260,10 +251,10 @@ jobs:
         env:
           RUNC_FLAVOR: ${{ matrix.runc }}
         run: |
-          sudo PATH=$PATH script/setup/install-seccomp
-          gosu root script/setup/install-runc
-          gosu root script/setup/install-cni
-          gosu root script/setup/install-critools
+          sudo -E PATH=$PATH script/setup/install-seccomp
+          sudo -E PATH=$PATH script/setup/install-runc
+          sudo -E PATH=$PATH script/setup/install-cni
+          sudo -E PATH=$PATH script/setup/install-critools
```

Comment:
quick question; I don't think we run on anything other than debian/ubuntu, but a comment on https://unix.stackexchange.com/a/83194/78656 mentioned that `sudo -E PATH=$PATH` didn't work on CentOS, but using `env PATH=$PATH` did